templates/p5p_metadata.erb

```diff
@@ -0,0 +1,12 @@
+set name=pkg.fmri value="<%=@name%>@<%=@version%>"
+set name=pkg.summary value="<%=@description%>"
+set name=pkg.human-version value="<%=@version%>"
+set name=pkg.description value="<%=@description%>"
+set name=info.classification value="org.opensolaris.category.2008:<%=@category%>"
+set name=variant.opensolaris.zone <%= attributes[:p5p_zonetype] %>
+set name=variant.arch value=<%=@architecture%>
+<transform file dir -> set owner <%= attributes[:p5p_user] %>>
+<transform file dir -> set group <%= attributes[:p5p_group] %>>
+<transform dir path=opt$->drop>
```

Comment:
These "drop" actions are to tell pkgmogrify to ignore the `set owner` commands, right? (I'm trying to eyeball the behavior without really knowing much about the p5p process)